packages/syft/src/syft/core/smpc/protocol/beaver/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""Beaver Generators Mechanism."""
+
+# relative
+from .beaver import *  # noqa F403
```

Comment:
Purely as a matter of style, we avoid using * imports because it can silenly overwrite things without our knowledge. Let's change this to explicit imports for each object being pulled in (one per line).

Reply:
Sure, I will change it :+1:

Reply:
Yeah @iamtrask , I actually wanted the code to executed , I removed `*` from it.